README.adoc

```diff
@@ -207,7 +167,7 @@ However, Cloud Spanner currently does not support cascading deletes on foreign k
 The dialect also supports unique column constraints applied through `@Column(unique = true)` or `@UniqueConstraint`.
 In these cases, the dialect will create a unique index to enforce uniqueness on the specified columns.
 
-==== Interleaved Tables
+==== Use Interleaved Tables for Parent-Child entities for better performance
```

Comment:
inconsistent capitalization?
```suggestion
==== Use Interleaved Tables for parent-child relationships for better performance
```

This could also go under performance section, or link back to it?

Reply:
Done. Also moved to performance section.